src/app/common/hooks/use-media-query.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
```

Comment:
Re-adding this which I naively removed as I thought we didn't need it, however it's important as it alters the breakpoint on re-size